### ISSUE TITLE
OSoC improve the UI

### ISSUE THREAD

**Issue by saarinii** [upes-open/near-me#8]
The UI built on flutter needs improve. The link for the design layout is mentioned below so try to match it as much as possible. Points for this issue will be provided screen-wise and completing 1 screen will get you 20 points. 
https://app.visily.ai/projects/f4ead457-ae47-422d-bc5c-bde02ee1f868/boards/511274

**Comment by Yash-Khattar** [comment]
Hey, I want to work on this issue. can you please assign it to me?

**Comment by saarinii** [comment]
@Charu-2718 @Yash-Khattar yes you guys can work on the issue. Please mention which screens you guys want to work on and distribute them accordingly. Whoever completes a screen first would get the points.

**Comment by Yash-Khattar** [comment]
@Charu-2718, Can I proceed with the launch and signup screen, you could start with others if it sounds good!

**Comment by Charu-2718** [comment]
Okay, I'll work on screens after that.

**Comment by Charu-2718** [comment]
#18

**Comment by Charu-2718** [comment]
#18 
Screens added

**Comment by saarinii** [comment]
> #18 Screens added

@Charu-2718 can you please share the ss of them as well ?

**Comment by saarinii** [comment]
I have updated the UI a bit so do check that out

**Comment by Charu-2718** [comment]
#25 
Added the screens according to the updated UI

**Comment by Charu-2718** [comment]
#26 
@saarinii Please check the Screens.

**Comment by Charu-2718** [comment]
#29 
@saarinii  All the screens are done, please check them once.
